packages/fx-core/resource/package.nls.json

```diff
@@ -293,10 +293,12 @@
   "core.createProjectQuestion.projectType.officeAddin.framework.placeholder": "Select a framework",
   "core.createProjectQuestion.projectType.tab.detail": "Embed your own web content in Teams, Outlook, and the Microsoft 365 app",
   "core.createProjectQuestion.projectType.tab.title": "App Features Using a Tab",
+  "core.createProjectQuestion.projectType.copilotExtension.label": "Copilot Extension",
+  "core.createProjectQuestion.projectType.copilotExtension.title": "App Features using Copilot Extension",
+  "core.createProjectQuestion.projectType.copilotExtension.detail": "Create declarative copilot, API plugin, or both for customized guidance and integration.",
```

Comment:
Revised: Create declarative Copilot, API plugin, or both with MS Copilot orchestrator and LLM

Note: This was the latest content. Confirm with Junjie.

Reply:
@MuyangAmigo are you OK the revised string proposed by Supriya?

Reply:
Yes, a minor thing is "MS" -> "Microsoft". Thanks!